projects/goharbor/harbor/docker/linux/harbor-portal/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nginx
 
-FROM node:16.10.0 as nodeportal
+FROM node:16.18.0 as nodeportal
```

Comment:
Can we use `https://gallery.ecr.aws/docker/library/node` instead of the node image from DockerHub?

Reply:
sure